fdbserver/TLogServer.actor.cpp

```diff
@@ -1513,12 +1513,12 @@ std::deque<std::pair<Version, LengthPrefixedStringRef>>& getVersionMessages(Refe
 	return tagData->versionMessages;
 };
 
-ACTOR Future<Void> waitForMessagesForTag(Reference<LogData> self, TLogPeekRequest const& req) {
-	auto tagData = self->getTagData(req.tag);
+ACTOR Future<Void> waitForMessagesForTag(Reference<LogData> self, TLogPeekRequest* req) {
```

Comment:
Changing from const-reference to pointer? Is there a deeper problem you are trying to solve with this change?

Reply:
you are not allowed to pass a const reference to an actor. Actors are always called by value. So the first version here didn't compile.